<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>0</number>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QSpinBox" name="speed">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;新宋体&quot;;</string>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pause">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>484</y>
          <width>91</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/mysource/pause.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="stop">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>550</y>
          <width>101</width>
          <height>71</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/mysource/stop.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="start">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>620</y>
          <width>111</width>
          <height>61</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>23</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/mysource/start.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="r0">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R0</string>
        </property>
       </widget>
       <widget class="QLabel" name="r1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R1</string>
        </property>
       </widget>
       <widget class="QLabel" name="r2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R2</string>
        </property>
       </widget>
       <widget class="QLabel" name="r3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R3</string>
        </property>
       </widget>
       <widget class="QLabel" name="r4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R4</string>
        </property>
       </widget>
       <widget class="QLabel" name="r5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R5</string>
        </property>
       </widget>
       <widget class="QLabel" name="r6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R6</string>
        </property>
       </widget>
       <widget class="QLabel" name="r7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R7</string>
        </property>
       </widget>
       <widget class="QLabel" name="r8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R8</string>
        </property>
       </widget>
       <widget class="QLabel" name="r9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R9</string>
        </property>
       </widget>
       <widget class="QLabel" name="r10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R10</string>
        </property>
       </widget>
       <widget class="QLabel" name="r11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R11</string>
        </property>
       </widget>
       <widget class="QLabel" name="r12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R12</string>
        </property>
       </widget>
       <widget class="QLabel" name="r13">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R13</string>
        </property>
       </widget>
       <widget class="QLabel" name="r14">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R14</string>
        </property>
       </widget>
       <widget class="QLabel" name="r15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>R15</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/mysource/bg.bmp);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="xinpian2440">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>330</y>
          <width>141</width>
          <height>121</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/mysource/arm.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="lamb" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>200</y>
          <width>51</width>
          <height>221</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="KEY0">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>172</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="KEY1">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="KEY2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>260</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="KEY3">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>300</y>
          <width>75</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="BEEP" name="BEEPS" native="true">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>260</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
       </widget>
       <widget class="LCDNUMBER" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>201</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="LCD" name="LCD_M" native="true">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>60</y>
          <width>221</width>
          <height>261</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1138</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>  文件    </string>
    </property>
    <widget class="QMenu" name="menu_bin">
     <property name="title">
      <string>打开bin文件</string>
     </property>
     <addaction name="actionnand"/>
     <addaction name="actionnor"/>
    </widget>
    <widget class="QMenu" name="menu_hex">
     <property name="title">
      <string>打开hex文件</string>
     </property>
     <addaction name="actionnand2"/>
     <addaction name="actionnor2"/>
    </widget>
    <addaction name="menu_bin"/>
    <addaction name="menu_hex"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>  芯片配置  </string>
    </property>
    <addaction name="actionAs"/>
    <addaction name="actionA"/>
    <addaction name="actionPc"/>
    <addaction name="actionIo"/>
   </widget>
   <widget class="QMenu" name="menuA">
    <property name="title">
     <string>  工具  </string>
    </property>
    <addaction name="run"/>
    <addaction name="stop_2"/>
    <addaction name="pause_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>  帮助  </string>
    </property>
    <addaction name="descption"/>
    <addaction name="about"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>  分析  </string>
    </property>
    <addaction name="actionM"/>
    <addaction name="actionR"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuA"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>300</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <action name="actionnand">
   <property name="text">
    <string>下载到nand Flash</string>
   </property>
  </action>
  <action name="actionnor">
   <property name="text">
    <string>下载到nor Flash</string>
   </property>
  </action>
  <action name="actionnand2">
   <property name="text">
    <string>下载到nand Flash</string>
   </property>
  </action>
  <action name="actionnor2">
   <property name="text">
    <string>下载到nor Flash</string>
   </property>
  </action>
  <action name="descption">
   <property name="text">
    <string>芯片手册</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="run">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="stop_2">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionAs">
   <property name="text">
    <string>nand Flash启动</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>nor Flash启动</string>
   </property>
  </action>
  <action name="pause_2">
   <property name="text">
    <string>暂停/运行</string>
   </property>
  </action>
  <action name="actionPc">
   <property name="text">
    <string>启动地址</string>
   </property>
  </action>
  <action name="actionM">
   <property name="text">
    <string>内存视图</string>
   </property>
  </action>
  <action name="actionR">
   <property name="text">
    <string>寄存器视图</string>
   </property>
  </action>
  <action name="actionIo">
   <property name="text">
    <string>外设配置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>lamb</class>
   <extends>QWidget</extends>
   <header>lamb.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BEEP</class>
   <extends>QWidget</extends>
   <header>beep.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LCDNUMBER</class>
   <extends>QWidget</extends>
   <header>lcdnumber.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LCD</class>
   <extends>QWidget</extends>
   <header location="global">lcd.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
